<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZCM_player</class>
 <widget class="QMainWindow" name="ZCM_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZCM_player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="Left_chennel">
      <property name="text">
       <string>Left chennel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="frame">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPlainTextEdit" name="zcm_event_info"/>
    </item>
    <item row="1" column="4">
     <widget class="QPlainTextEdit" name="zcm_info"/>
    </item>
    <item row="0" column="0" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="zcm_event_scale">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="zcm_event_num">
        <property name="maximum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zcm_event_size">
        <property name="text">
         <string>from 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZCM_player">
    <property name="title">
     <string>ZCM player</string>
    </property>
   </widget>
   <addaction name="menuZCM_player"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
